std_msgs/Header header
nav_msgs/MapMetaData info
int16[] front_x
int16[] front_y
int16[] unex_x
int16[] unex_y
int16[] obb_x
int16[] obb_y